### PR TITLE
Document how to do hero blocks in Whitehall

### PR DESCRIPTION
**Pull request by richardTowers**

### CODE REVIEW

File: docs/building_blocks_for_flexible_content.md [hunk 1]
```diff
@@ -320,16 +325,28 @@ A hero block renders a vertical column of blocks in a two-thirds-width block on
...
-        text: "Learn more about our goals"
-        href: "/landing-page/goals"
 ```
 
+When authored in Whitehall, you should upload the images and then refer to them using their markdown syntax as follows:
+
+```yaml
+- type: hero
+  image:
+    alt: "Placeholder alt text"
```
Comment (maxgds) [inline docs/building_blocks_for_flexible_content.md:335]: I thought we agreed no alt text? Hero should always be decor so we shouldn't supply this.

File: docs/building_blocks_for_flexible_content.md [hunk 2]
```diff
@@ -305,6 +305,11 @@ A hero block renders a vertical column of blocks in a two-thirds-width block on
 
 ##### Example
 
+The YAML structure for hero images is slightly different depending on whether you're looking at the content item in Content Store (or a hardcoded YAML block which has to match the Content Store format),
+or in Whitehall.
```
Comment (maxgds) [inline docs/building_blocks_for_flexible_content.md:309]: nitpick: unnecessary newline?

File: docs/building_blocks_for_flexible_content.md (hunk 1 above)
Comment (richardTowers) [reply docs/building_blocks_for_flexible_content.md:335]: Yep, this needs removing from the example - sorry!

Comment (richardTowers) [reply docs/building_blocks_for_flexible_content.md]: Fixed in 6e09625b8532b9235a9a105c960abda5a453ab93

File: docs/building_blocks_for_flexible_content.md (hunk 2 above)
Comment (richardTowers) [reply docs/building_blocks_for_flexible_content.md]: Fixed in the force push (the line was just a bit too long for my tastes, but happy to have it on one line)

**Review by maxgds — APPROVED** [review]

**Pull request merged**